src/lib/index.ts

```diff
@@ -38,8 +38,15 @@ class TPAStylePlugin {
     );
     this.replaceRuntimeModule(compiler);
 
+    const pluginDescriptor = isWebpack5
+      ? TPAStylePlugin.pluginName
+      : {
+          name: TPAStylePlugin.pluginName,
+          stage: compilation.PROCESS_ASSETS_STAGE_OPTIMIZE,
+        };
```

Comment:
This new hook now requires a `stage` enum.

Reply:
This is similar to how its done in other plugins: https://github.com/NMFR/last-call-webpack-plugin/pull/9/files.